src/fetch.ts

```diff
@@ -77,7 +77,9 @@ export function createFetch ({ fetch }: CreateFetchOptions): $Fetch {
         request = withQuery(request, opts.params)
       }
       const hasPayload = payloadMethods.includes((opts.method || '').toLowerCase())
-      if (opts.body && typeof opts.body === 'object' && hasPayload) {
+      const supportsFormData = typeof FormData === 'function'
+      const isFormData = supportsFormData && opts.body instanceof FormData
```

Comment:
We cannot rely on global FormData detection. One can directly use a package like [form-data](https://www.npmjs.com/package/form-data) in Node.js without global polyfill.